lib/perl/Genome/Model/Tools/DetectVariants2/SpeedseqSv.t

```diff
@@ -36,7 +36,7 @@ symlink($reference_fasta_index,$reference_build->data_directory .'/all_sequences
 
 my $result_users = Genome::Test::Factory::SoftwareResult::User->setup_user_hash(reference_sequence_build_id => $reference_build->id,);
 
-my $params = "-g,-d,-o:Hello";
+my $params = '-P,-g,-d';
```

Comment:
The latest commit adds a colon separated (not semi-colon as the commit suggests) param that is parsed and used in the Speedseq SV tool.